packages/airnode-node/test/setup/e2e/testing.ts

```diff
@@ -20,7 +21,13 @@ export const deployAirnodeAndMakeRequests = async (filename: string, requests?:
   });
   // TODO: This is caused by duplicated mnemonic in Airnode state
   (config.nodeSettings as any).airnodeWalletMnemonic = deployConfig.airnodes.CurrencyConverterAirnode.mnemonic;
-  jest.spyOn(fs, 'readFileSync').mockReturnValue(JSON.stringify(config));
+  jest.spyOn(fs, 'readFileSync').mockImplementation((...args) => {
```

Comment:
@Siegrift Your solution works everywhere so I've updated this to use `mockImplementation`.